From 735927c7a72b69f60f13fe7bcf4a10c1a80ca151 Mon Sep 17 00:00:00 2001
From: Robby Workman <rworkman@slackware.com>
Date: Thu, 19 Feb 2015 19:07:06 -0600
Subject: [PATCH] icons/Makefile.am: Remove unnecessary gtk-update-icon-cache

No icons are installed in the standard paths any more, so this
appears to be leftover code.
---
 icons/Makefile.am | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/icons/Makefile.am b/icons/Makefile.am
index 70638c1..035476a 100644
--- a/icons/Makefile.am
+++ b/icons/Makefile.am
@@ -4,17 +4,4 @@ SUBDIRS =								\
 	16x16								\
 	scalable
 
-gtk_update_icon_cache = gtk-update-icon-cache -f -t $(datadir)/icons/hicolor
-
-install-data-hook:
-	@-if test -z "$(DESTDIR)"; then					\
-                echo "Updating Gtk icon cache.";			\
-                $(gtk_update_icon_cache);				\
-        else								\
-                echo "*** Icon cache not updated. Remember to run:";	\
-		echo "***";						\
-                echo "***   $(gtk_update_icon_cache)";			\
-		echo "***";						\
-        fi
-
 # vi:set ts=8 sw=8 noet ai nocindent syntax=automake:
-- 
2.3.1

